src/cobra/test/test_core/test_core_reaction.py

```diff
@@ -17,7 +17,18 @@ def test_gpr() -> None:
     """Test GPR evaluation."""
     model = Model()
     reaction = Reaction("test")
-
+    reaction.update_genes_from_gpr()
```

Comment:
Why does that need to be called here. Isn't the default GPR empty?

Reply:
I think it is an attempt to test lines 418-419 of reaction.py, but it didn't seem to work